CHANGELOG.md

```diff
@@ -19,6 +19,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 
+## [7.1.2](https://github.com/ruflin/Elastica/compare/7.1.1...7.1.2)
```

Comment:
This should be removed as it's done upper. Proper version will be edited when preparing the next release.